get-message-service-bus-queue-peek-lock-renew-sequential-convoy/manifest.json

```
@@ -1,9 +1,9 @@
 {
-    "title": "Azure Service Bus: Get message from a queue with token renewal - Peek lock renew pattern",
+    "title": "Azure Service Bus: Sequential convoy pattern",

```

This title appears on the gallery browse page and should be more descriptive for customers who don't know what "sequential convoy" means as well as those who do. Also use word structure similar to other template titles (verb-first wording).

```suggestion
    "title": "Azure Service Bus: Process related messages from a session-enabled queue for the same workflow instance - Sequential convoy pattern",

```